codes/15486_py3_sjskgks.py

```diff
@@ -0,0 +1,14 @@
+import sys
+input = sys.stdin.readline
+
+last_work = int(input())
+max_reward = [0 for _ in range(last_work+1)]
+max_day = 0 # 일 시작하는 날 전까지 중 최대 값

```

Comment:
네이밍을 조금 더 좋게 할 수 있을 것 같습니다.
```suggestion
rewards = [0 for _ in range(last_work+1)]
max_reward = 0
```